tailwind.config.js

```diff
@@ -1,8 +1,27 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,ts}"],
+  content: ['./src/**/*.{html,ts}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: '#222328',
+        secondary: '#969AB0',
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+    },
+    fontFamily: {
+      sans: ['IBM Plex Sans', 'ui-sans-serif', 'system-ui'],
```

Comment:
Will this apply your font by default, or you need to add 'font-sans' everywhere?

Reply:
agree, you need to apply fonts as a default global fonts for your project

Reply:
resolved